backend/commands/import_attendance_list.py

```diff
@@ -65,10 +71,13 @@ def import_attendance_list():
 
     # assign attendance list
     for participant_idx, row in enumerate(rows, 1):
+        participant = Participant.query.get(participant_idx)
         for hacknight_index, element in enumerate(row[5:], 1):
-            participant = Participant.query.get(participant_idx)
-            if "1" in element:
+            if "1" in element and participant:
```

Comment:
sometimes `participant` is completely empty, no idea why, I just ignored such cases